clients/admin-ui/src/features/common/tablev2/EmptyTableState.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, HStack, Text, WarningTwoIcon } from "@fidesui/react";
```

Comment:
can we namespace these files as `table/v2` as opposed to `tablev2` to mimic what we already have for `nav/v2`?

Reply:
we can. good idea!